internal/jsre/deps/web3.js

```diff
@@ -5396,8 +5396,8 @@ var methods = function () {
     var estimateGas = new Method({
         name: 'estimateGas',
         call: 'eth_estimateGas',
-        params: 1,
-        inputFormatter: [formatters.inputCallFormatter],
+        params: 2,
+        inputFormatter: [formatters.inputCallFormatter, formatters.inputDefaultBlockNumberFormatter],
```

Comment:
If you set `nil` here, will the previous behaviour be the defautl? as in 
```
 inputFormatter: [formatters.inputCallFormatter, nil],
```

Reply:
It wouldn't, but as it turns out you don't need to change the web3.js